### ISSUE TITLE
Tray doesn't change on build

### ISSUE THREAD

**Issue by nvlled** [neutralinojs/neutralinojs#1115]
**Describe the bug**
Calling os.setTray()  on development setup (with neu run) changes the tray just fine. But on build (running the binary directly), the tray doesn't change after first call.

**To Reproduce**
1. Run app with neu run
2. Call os.setTray() several times with different icons
3. Note how the icon changes successfully
4. Run neu build and run binary
5. Call os.setTray()  several times with different icons
6. Note the icon changes only once, and succeeding calls has no effect

**Expected behavior**
os.setTray() should change the icon for each invocation.

**Specifications**  
 - OS: [e.g. Linux x64]
 - Neutralinojs version:  v4.11.0
 - Neutralinojs client library version: v3.9.0
 - Neutralinojs CLI version: v9.5.1

**Additional context**
I made sure to delay and await for each os.setTray() calls.

**Comment by nvlled** [comment]
I don't know much C++, but after some poking around, I think the problem is around here
https://github.com/neutralinojs/neutralinojs/blob/main/api/os/os.cpp#L598-L602
Specifically, this part: `resources::extractFile(iconPath, tempIconPath);` I'm guessing `extractFile()` caches the read, so it's ends up returning the same icon afterwards.

**Comment by nvlled** [comment]
Okay, after some poking around the C++ codebase, I figured out the problem.

The main problem is that `tray_update()` actually does nothing when it is passed with the same icon path as before, even though the actual icon file may have changed. In the bundle resource mode, it uses the same [temporary filename every time](https://github.com/neutralinojs/neutralinojs/blob/main/api/os/os.cpp#L598-L602), causing `tray_update()` to only change once.

I made some changes so that it alternates between two different temporary filenames, and this fixes the problem, as hypothesized. I don't know if this is the best solution, but I'll open a pull request anyways.
